rkt/images.go

```diff
@@ -408,6 +408,18 @@ func (f *fetcher) fetch(appName string, aciURL, ascURL string, ascFile *os.File,
 		return nil, aciFile, nil, nil
 	}
 
+	// attempt to automatically fetch the public key in case it is available on a TLS connection.
+	if !globalFlags.InsecureSkipVerify && appName != "" {
+		pkls, err := getPubKeyLocations(appName, false, globalFlags.Debug)
+		if err != nil {
+			stderr("Error determining key location: %v", err)
+		} else {
+			if err := addKeys(pkls, appName, false, true, false); err != nil {
```

Comment:
I'm not sure if you are going to actually replace the bools in the prototype of this function to some enum values. If not, then please add some comment what these values do. Like "no http, don't ask user for accepting the key, no overriding". Or something like that.